### PR TITLE
Upgrade axios & fix tests

### PR DESCRIPTION
**Pull request by tirumerla**

### CODE REVIEW

File: examples/batch-proposal/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "example-batch-proposal",
-  "version": "1.46.0",
+  "version": "1.47.0-rc.0",
   "private": true,
   "main": "index.js",
   "author": "Ernesto García <mverzilli@openzeppelin.com>",
```
Comment (shahnami) [inline examples/batch-proposal/package.json:6]: Just noticed this dual identity @ernestognw 😆

File: packages/relay/package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "web3-core-helpers": "^1.8.2"
   },
   "dependencies": {
-    "@openzeppelin/defender-base-client": "1.46.0",
+    "@openzeppelin/defender-base-client": "1.47.0-rc.0",
     "amazon-cognito-identity-js": "^6.0.1",
     "axios": "^0.21.2",
```
Comment (shahnami) [inline packages/relay/package.json:36]: Same here?

File: packages/sentinel/package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
   "license": "MIT",
   "dependencies": {
     "@ethersproject/abi": "^5.6.3",
-    "@openzeppelin/defender-base-client": "1.46.0",
+    "@openzeppelin/defender-base-client": "1.47.0-rc.0",
     "axios": "^0.21.2",
```
Comment (shahnami) [inline packages/sentinel/package.json:26]: Same here?

File: examples/batch-proposal/package.json (hunk 1 above)
Comment (ernestognw) [reply examples/batch-proposal/package.json]: You gotta see me with my hair tied up. @dylankilkenny can confirm

**Comment by tirumerla** [comment]
> **🚨 Potential security issues detected.** Learn more about [Socket for GitHub ↗︎](https://socket.dev?utm_medium=gh)
> 
> To accept the risk, merge this PR and you will not be notified again.
> 
> Issue	Package	Version	Note	Source
> [ Install scripts ](https://socket.dev/npm/package/nx/issues/16.5.1?issue=installScripts)	[ nx ](https://socket.dev/npm/package/nx/overview/16.5.1)	[ 16.5.1 ](https://socket.dev/npm/package/nx/files/16.5.1)	
> * **Install script:** [postinstall](https://socket.dev/npm/package/nx/files/16.5.1/package.json#T350-375)
> * **Source:** `node ./bin/post-install`
> 
> * [lerna@6.6.2](https://socket.dev/npm/package/lerna/overview/6.6.2) via [`package.json`](https://github.com/OpenZeppelin/defender-client/pull/286/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) [`package.json`](https://github.com/OpenZeppelin/defender-client/pull/286/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)
> 
> ### Next steps
> ** What is an install script? **
> ** Take a deeper look at the dependency **
> Take a moment to review the security alert above. Review the linked package source code to understand the potential risk. Ensure the package is not malicious before proceeding. If you're unsure how to proceed, reach out to your security team or ask the Socket team for help at support [AT] socket [DOT] dev.
> 
> ** Remove the package **
> If you happen to install a dependency that Socket reports as [Known Malware](https://socket.dev/npm/issue/malware) you should immediately remove it and select a different dependency. For other alert types, you may may wish to investigate alternative packages or consider if there are other ways to mitigate the specific risk posed by the dependency.
> 
> **Mark a package as acceptable risk**
> To ignore an alert, reply with a comment starting with `@SocketSecurity ignore` followed by a space separated list of `package-name@version` specifiers. e.g. `@SocketSecurity ignore foo@1.0.0 bar@*` or ignore all packages with `@SocketSecurity ignore-all`
> 
> * `@SocketSecurity ignore nx@16.5.1`

@SocketSecurity ignore nx@16.5.1

**Comment by tirumerla** [comment]
@SocketSecurity ignore nx@16.5.1

**Pull request merged**